src/solver/application.cpp

```diff
@@ -375,7 +372,7 @@ void Application::readDataForTheStudy(Data::StudyLoadOptions& options)
     }
 
     // Errors
-    if (pErrorCount || pWarningCount || study.gotFatalError)
+    if (pErrorCount || pWarningCount)
```

Comment:
In some cases, this will never be run (see l. 352). This is a great drawback IMO